di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -119,21 +119,27 @@ public SharedClaims mapToSharedClaim(Identity identity, boolean agedDOB) {
                                 })
                         .collect(toList());
 
+        List<NameParts> namePartsList = new ArrayList<>();
+        if (identity.name().firstName() != null)
+            namePartsList.add(new NameParts(GIVEN_NAME, identity.name().firstName()));
+
+        if (identity.name().middleName() != null)
+            namePartsList.add(new NameParts(GIVEN_NAME, identity.name().middleName()));
```

Comment:
Can we also have a `!isBlank` check too please? I have recently found that it will populate the DB with a "GivenName" that has no value. I recently added this so you'll see the change on rebase.

Reply:
This change was to solve that exact issue (given name with no value), but it appears your recent change has prevent that from occurring. So this code can be removed.